mpi-proxy-split/lower-half/kernel_loader.cpp

```diff
@@ -119,22 +116,6 @@ int main(int argc, char *argv[], char *envp[]) {
   memset(lh_info, 0, sizeof(LowerHalfInfo_t));
   lh_info->fsaddr = (void*)fsaddr;
  
```

Comment:
I forget the details here.  I remmeber that both DMTCP and MANA were creating a memory segment just above the end of data (above `sbrk(0)`).  This was to make sure that libc.so doesn't try to extend thee alloc arena, so as to simplify the logic.

I'm assuming that DMTCP does it now, and MANA doesn't need to now?  Is that correct?  Could you reply here on github, so that we have a permanent record of our thinking?

Reply:
For now, DMTCP, MANA's upper half, and the kernel loader all have this functionality. I think it's safe to remove the corresponding parts in MANA's upper half and the kernel loader. But I haven't tested. We can do this later.